terraform/amazon/templates/bastion_user_data.yaml.template

```diff
@@ -73,13 +78,15 @@ write_files:
         chmod 755 /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy ;\
       fi ;\
       /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy download --output /opt/wing-$${WING_VERSION}/wing --sha256sums https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/tarmak_$${WING_VERSION}_checksums.txt  --signature-armored https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/tarmak_$${WING_VERSION}_checksums.txt.asc https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/wing_$${WING_VERSION}_linux_amd64'
+ {{- end }}
     ExecStartPre=/bin/sh -c '\
       set -e ;\
+      usermod -a -G ssh_keys wing ;\
```

Comment:
Is that really necessary to read the public host key?

Reply:
This is so wing can read the ssh private keys to sign the document, to prove we have them